pack.yaml

```
@@ -1,16 +1,16 @@
 ---
 ref: rabbitmq
-name : rabbitmq
-description : RabbitMQ integration
+name: rabbitmq
+description: RabbitMQ integration
 keywords:
   - rabbitmq
   - queuing
   - messaging
   - aqmp
   - stomp
   - message broker
-version: 1.0.0
+version: 1.0.1
```

```suggestion
version: 1.1.0
```
Let's bump the minor version for the new feature, as patch version is only for bugfixes.